src/Raven.Server/Documents/Includes/IncludeTimeSeriesCommand.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Collections.Generic;
+using Raven.Client.Documents.Operations.TimeSeries;
+using Raven.Client.Documents.Session;
+using Raven.Server.Documents.Handlers;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Includes
+{
+    public class IncludeTimeSeriesCommand
+    {
+        private readonly DocumentDatabase _database;
+        private readonly DocumentsOperationContext _context;
+        private readonly Dictionary<string, (IList<string> TimeseriesNames, IList<string> FromList, IList<string> ToList)> _timeSeriesBySourcePath;
+        private readonly Dictionary<string, HashSet<TimeSeriesRange>> _timeSeriesRangesBySourcePath;
+
+
+        public Dictionary<string, List<TimeSeriesRangeResult>> Results { get; }
+
+        public IncludeTimeSeriesCommand(DocumentDatabase database, DocumentsOperationContext context)
+        {
+            _database = database;
+            _context = context;
+
+            Results = new Dictionary<string, List<TimeSeriesRangeResult>>(StringComparer.OrdinalIgnoreCase);
+        }
+
+        public IncludeTimeSeriesCommand(DocumentDatabase database, DocumentsOperationContext context, IList<string> timeseriesNames, IList<string> fromList, IList<string> toList) 
+            : this(database, context)
+        {
```

Comment:
Need to validate here that we got the same number of time series, from and to

Reply:
done